### ISSUE TITLE
why you decrease  kl_loss weight from 1.0 (vits) to 0.1 ? 

### ISSUE THREAD

**Issue by segmentationFaults** [sh-lee-prml/HierSpeechpp#25]

**Comment by segmentationFaults** [comment]
Can you share your insight?

**Comment by Gabibing** [comment]
I'm not the author. but, in my opinion, when the difference between the distributions is large, the KL loss can be higher. Trying to make the distributions identical by assigning a high weight might actually lead to a bad result.

**Comment by sh-lee-prml** [comment]
Hi! @segmentationFaults 

As @Gabibing said, we observed that the model could not reconstruct the wav2vec representation with the weight of 1. (However, this model also could synthesize the speech from text!)

However, I acknowledge that using the weight of 1 is natural in terms of Normalizing Flow and Text Encoder (The weights of both modules are optimized only with kl loss. 

In my case, I hope to improve the reconstruction quality so I simply decrease the weight of kl loss. 

For LibriTTS dataset, we found that using the weight of 0.1 has a better performance than that of 1, 0.5.

Thanks!

and Thanks @Gabibing
